routes/project.js

```diff
@@ -182,6 +182,7 @@ router.put('/edit/:id', authCheck, (req, res) => {
   Project.findByIdAndUpdate(id, project)
     .then(() => {
       return res.status(200).json({
+        id,
```

Comment:
you can remove the id. You don't assign any value to it, so it is not serialized in the response.

Reply:
The value of project._id is assigned to id, because the next function(delete) require id.

Reply:
Ok.